tests/frameworks/OSXShims/include/windows.h

```diff
@@ -25,6 +25,10 @@
 #define _MAX_PATH PATH_MAX
 #endif
 
+#ifndef MAX_PATH
+#define MAX_PATH PATH_MAX
+#endif
```

Comment:
The actual constant on windows is `MAX_PATH`.

---
In reply to: [89023623](https://github.com/Microsoft/WinObjC/pull/1438#discussion_r89023623) [](ancestors = 89023623)